prow/repoowners/repoowners.go

```diff
@@ -171,7 +174,12 @@ func (c *Client) LoadRepoOwners(org, repo string) (*RepoOwners, error) {
 			// aliases must be loaded
 			entry.aliases = loadAliasesFrom(gitRepo.Dir, log)
 		}
-		entry.owners, err = loadOwnersFrom(gitRepo.Dir, mdYaml, entry.aliases, log)
+
+		var dirBlacklist []string
+		if getter := c.DirBlacklist; getter != nil {
```

Comment:
Might prefer a base impl of the getter that return `[]string{}` instead of having to do a `nil` check here